applications/vanilla/controllers/api/CommentsApiController.php

```diff
@@ -184,6 +184,57 @@ public function get($id, array $query) {
         return $result;
     }
 
+    /**
+     * Get a comments quote data.
+     *
+     * @param int $id The ID of the comment.
+     *
+     * @return array The comment quote data.
+     *
+     * @throws NotFoundException If the record with the given ID can't be found.
+     * @throws \Exception if no session is available.
+     * @throws \Vanilla\Exception\PermissionException if the user does not have the specified permission(s).
+     * @throws \Garden\Schema\ValidationException If the output schema is configured incorrectly.
+     */
+    public function get_quote($id) {
+        $this->permission();
```

Comment:
Should probably include a permission here, even if it's just Garden.SignIn.Allow.